portfolio/src/main/webapp/style.css

```diff
@@ -0,0 +1,44 @@
+body{
+    margin:0;
```

Comment:
update the css similarly to the suggestions in https://github.com/samuelsogbesan/step/pull/7/files/a37e16b4f33284cca5f9c0ce40d45b209e79ae64#diff-680ee8528519da343f72308f826ee0e1

Reply:
- Updated tab spacing to +2 in commit 609c261
-  Add spacing between '{' and selector in 396d0c4